tool/plugin/lib/plugin.dart

```diff
@@ -525,7 +525,11 @@ class BuildCommand extends ProductCommand {
 
       // Handle skipped files.
       for (var file in spec.filesToSkip) {
```

Comment:
make this line
var (String file in spec.filesToSkip) {
as the type isn't being inferred.

Reply:
Will do!

Reply:
Wait, if the type isn't being inferred then how does this even work?